hmailserver/installation/section_files_64.iss

```diff
@@ -7,6 +7,11 @@ Source: "Microsoft.VC142.CRT\*"; DestDir: "{app}\Bin"; Flags: ignoreversion; Com
 
```

Comment:
Is there some innosetup-magic which prevents this line from installing the Windows-*KB*-msu's into the bin-directory (and not just into the temp directory)?

Reply:
> Is there some innosetup-magic which prevents this line from installing the Windows-_KB_-msu's into the bin-directory (and not just into the temp directory)?

No. Hmm, was thinking i changed it to Temp.
but of cause, we should change it to Temp!

Reply:
Do you intend to update the pull request with this or do you want me to fix it after merge?